Sources/FoundationEssentials/Calendar/Calendar_Gregorian.swift

```diff
@@ -37,7 +37,7 @@ extension Date {
     }
 
     var julianDay: Int {
-        Int(julianDate.rounded())
+        Int((julianDate + 0.5).rounded(.down))
```

Comment:
Is there a behavioral difference between these two? `.rounded()` at face value seems simpler but perhaps I'm not thinking of the right number that would differ in behavior here

Reply:
The default rounding behavior for `rounded()` is even or away from zero. They behave differently if `julianDate` is negative.

For example, if `julianDate == -0.5` 
```
Int(julianDate.rounded()) == -1 // wrong
Int((julianDate + 0.5).rounded(.down)) == 0 // correct
```

What this is doing to match a floating point `julianDate: Double` (a point in time) to a `julianDay: Int` to represent a calendar day (a span of time). 

I find it easier to start with a positive day number, so here's an example

Given `julianDate: 2451545.0 == 2000-01-01 12:00:00` and its `julianDay == 2451545`, find some function `fn` so that every point in calendar day 2000-01-01 has `julianDay == 2451545`

So:
```
fn(2451544.5..<2451545.5) -> 2451545

// Literally
fun(2000-01-01 00:00:00 ..< 2000-01-02 00:00:00) -> 2000-01-01
```

Similarly for negative Julian days:
```
fn(-0.5..<0.5) -> 0

// Literally
fun(-4712-01-01 00:00:00 ..< -4712-01-02 00:00:00) -> -4712-01-01 
```

That is why `-0.5` needs to be rounded to `0`, not `-1`

Reply:
Ah I see, makes sense - thanks for the explanation!